diff -urNp minitube-1.9-orig/src/qtsingleapplication/qtlocalpeer.cpp minitube-1.9/src/qtsingleapplication/qtlocalpeer.cpp
--- minitube-1.9-orig/src/qtsingleapplication/qtlocalpeer.cpp	2012-09-27 10:17:03.000000000 +0000
+++ minitube-1.9/src/qtsingleapplication/qtlocalpeer.cpp	2013-02-19 00:12:22.371294770 +0000
@@ -68,6 +68,7 @@ namespace QtLP_Private {
 #include "qtlockedfile_unix.cpp"
 #endif
 }
+#include <unistd.h>
 
 const char* QtLocalPeer::ack = "ack";
 
@@ -101,7 +102,8 @@ QtLocalPeer::QtLocalPeer(QObject* parent
         socketName += QLatin1Char('-') + QString::number(sessionId, 16);
     }
 #else
-    socketName += QLatin1Char('-') + QString::number(::getuid(), 16);
+    int uid=QtLP_Private::getuid();
+    socketName += QLatin1Char('-') + QString::number(uid, 16);
 #endif
 
     server = new QLocalServer(this);
