diff -urNp pingus-0.7.6-orig/SConscript pingus-0.7.6/SConscript
--- pingus-0.7.6-orig/SConscript	2011-12-24 21:46:47.000000000 +0000
+++ pingus-0.7.6/SConscript	2013-03-23 22:18:05.706685863 +0000
@@ -188,7 +188,7 @@ class Project:
             
     def configure_boost(self):
         if not self.conf.CheckLibWithHeader('boost_signals', 'boost/signals.hpp', 'c++'):
-            if not self.conf.CheckLibWithHeader('boost_signals-mt', 'boost/signals.hpp', 'c++'):
+            if not self.conf.CheckLibWithHeader('boost_signals', 'boost/signals.hpp', 'c++'):
                 self.fatal_error += "  * library 'boost_signals' not found\n"
 
     def configure_png(self):
diff -urNp pingus-0.7.6-orig/src/pingus/pingus_main.cpp pingus-0.7.6/src/pingus/pingus_main.cpp
--- pingus-0.7.6-orig/src/pingus/pingus_main.cpp	2011-12-24 21:46:47.000000000 +0000
+++ pingus-0.7.6/src/pingus/pingus_main.cpp	2013-03-23 21:49:49.527641894 +0000
@@ -332,7 +332,7 @@ PingusMain::parse_args(int argc, char**
 
       case 'V':
         std::cout <<
-          "Pingus " VERSION "\n"
+          "Pingus 0.7.6\n"
           "Copyright (C) 1998-2011 Ingo Ruhnke <grumbel@gmail.com>\n"
           "See the file AUTHORS for a complete list of contributors.\n"
           "Pingus comes with ABSOLUTELY NO WARRANTY. This is free software, and you are\n"
@@ -465,7 +465,7 @@ PingusMain::init_path_finder()
 void
 PingusMain::print_greeting_message()
 {
-  std::string greeting = "Welcome to Pingus "VERSION;
+  std::string greeting = "Welcome to Pingus 0.7.6";
   greeting += "!";
   std::cout <<  greeting << std::endl;
   for (unsigned int i = 0; i < greeting.length(); ++i)
diff -urNp pingus-0.7.6-orig/src/pingus/screens/pingus_menu.cpp pingus-0.7.6/src/pingus/screens/pingus_menu.cpp
--- pingus-0.7.6-orig/src/pingus/screens/pingus_menu.cpp	2011-12-24 21:46:47.000000000 +0000
+++ pingus-0.7.6/src/pingus/screens/pingus_menu.cpp	2013-03-23 21:53:36.040647748 +0000
@@ -153,7 +153,7 @@ PingusMenu::draw_background(DrawingConte
                          gc.get_height()/2 - 280));
 
   gc.print_left(Fonts::pingus_small, Vector2i(gc.get_width()/2 - 400 + 25, gc.get_height()-140),
-                "Pingus "VERSION" - Copyright (C) 1998-2011 Ingo Ruhnke <grumbel@gmail.com>\n"
+                "Pingus 0.7.6 - Copyright (C) 1998-2011 Ingo Ruhnke <grumbel@gmail.com>\n"
                 "See the file AUTHORS for a complete list of contributors.\n"
                 "Pingus comes with ABSOLUTELY NO WARRANTY. This is free software, and you are\n"
                 "welcome to redistribute it under certain conditions; see the file COPYING for details.\n");
