diff -urNp pygame-1.9.1release-orig/config_unix.py pygame-1.9.1release/config_unix.py
--- pygame-1.9.1release-orig/config_unix.py	2009-05-26 21:15:24.000000000 +0000
+++ pygame-1.9.1release/config_unix.py	2012-12-27 21:18:06.439955010 +0000
@@ -19,6 +19,11 @@ origincdirs = ['/include', '/include/SDL
                '/include/smpeg' ]
 origlibdirs = ['/lib','/lib64','/X11R6/lib']
 
+kanapi_inc = os.environ.get('KANAPI_INC', '')
+kanapi_lib = os.environ.get('KANAPI_LIB', '')
+origincdirs = kanapi_inc.split(":")
+origlibdirs = kanapi_lib.split(":")
+
 def confirm(message):
     "ask a yes/no question, return result"
     reply = raw_input('\n' + message + ' [Y/n]:')
@@ -144,7 +149,7 @@ def main():
         DependencyProg('SMPEG', 'SMPEG_CONFIG', 'smpeg-config', '0.4.3', ['smpeg']),
         Dependency('PNG', 'png.h', 'libpng', ['png']),
         Dependency('JPEG', 'jpeglib.h', 'libjpeg', ['jpeg']),
-        Dependency('SCRAP', '', 'libX11', ['X11']),
+        Dependency('SCRAP', 'X11/Xlib.h', 'libX11', ['X11']),
         Dependency('PORTMIDI', 'portmidi.h', 'libportmidi.so', ['portmidi']),
         Dependency('PORTTIME', 'porttime.h', 'libporttime.so', ['porttime']),
         #Dependency('GFX', 'SDL_gfxPrimitives.h', 'libSDL_gfx.so', ['SDL_gfx']),
@@ -157,12 +162,12 @@ def main():
         incdirs = [localbase+d for d in origincdirs]
         libdirs = [localbase+d for d in origlibdirs]
     else:
-        incdirs = []
-        libdirs = []
-    incdirs += ["/usr"+d for d in origincdirs]
-    libdirs += ["/usr"+d for d in origlibdirs]
-    incdirs += ["/usr/local"+d for d in origincdirs]
-    libdirs += ["/usr/local"+d for d in origlibdirs]
+        incdirs = origincdirs
+        libdirs = origlibdirs
+#    incdirs += ["/usr"+d for d in origincdirs]
+#    libdirs += ["/usr"+d for d in origlibdirs]
+#    incdirs += ["/usr/local"+d for d in origincdirs]
+#    libdirs += ["/usr/local"+d for d in origlibdirs]
 
     for arg in DEPS[0].cflags.split():
         if arg[:2] == '-I':
diff -urNp pygame-1.9.1release-orig/Setup.in pygame-1.9.1release/Setup.in
--- pygame-1.9.1release-orig/Setup.in	2009-07-02 04:41:56.000000000 +0000
+++ pygame-1.9.1release/Setup.in	2012-12-27 21:20:27.053197146 +0000
@@ -34,7 +34,7 @@ _numericsurfarray src/_numericsurfarray.
 _numericsndarray src/_numericsndarray.c $(SDL) $(MIXER) $(DEBUG)
 movie src/movie.c $(SDL) $(SMPEG) $(DEBUG)
 scrap src/scrap.c $(SDL) $(SCRAP) $(DEBUG)
-_camera src/_camera.c src/camera_v4l2.c src/camera_v4l.c $(SDL) $(DEBUG)
+#_camera src/_camera.c src/camera_v4l2.c src/camera_v4l.c $(SDL) $(DEBUG)
 pypm src/pypm.c $(SDL) $(PORTMIDI) $(PORTTIME) $(DEBUG)
 
 GFX = src/SDL_gfx/SDL_gfxPrimitives.c 
