diff -urNp xpaint-2.9.9.1-orig/app-defaults/out/XPaint xpaint-2.9.9.1/app-defaults/out/XPaint
--- xpaint-2.9.9.1-orig/app-defaults/out/XPaint	2012-02-08 11:51:15.000000000 +0100
+++ xpaint-2.9.9.1/app-defaults/out/XPaint	2012-12-16 22:24:47.224325621 +0100
@@ -13,7 +13,7 @@ xpaint*msgFile:  messages/Messages
 xpaint*menufont: Liberation-10:matrix=0.85 0 0 0.9
 
 ! Xpaint share directory
-xpaint*shareDir: /usr/share/xpaint
+xpaint*shareDir: /kanapi_packages/xpaint-2.9.9.1/share/xpaint
 
 !  This fixes Xpaint's startup features
 !xpaint*horizontal: True
diff -urNp xpaint-2.9.9.1-orig/app-defaults/out/XPaint_es xpaint-2.9.9.1/app-defaults/out/XPaint_es
--- xpaint-2.9.9.1-orig/app-defaults/out/XPaint_es	2012-02-08 11:51:15.000000000 +0100
+++ xpaint-2.9.9.1/app-defaults/out/XPaint_es	2012-12-16 22:24:37.580276712 +0100
@@ -13,7 +13,7 @@ xpaint*msgFile:  messages/Messages_es
 xpaint*menufont: Liberation-10:matrix=0.85 0 0 0.9
 
 ! Xpaint share directory
-xpaint*shareDir: /usr/share/xpaint
+xpaint*shareDir: /kanapi_packages/xpaint-2.9.9.1/share/xpaint
 
 !  This fixes Xpaint's startup features
 !xpaint*horizontal: True
diff -urNp xpaint-2.9.9.1-orig/app-defaults/out/XPaint_fr xpaint-2.9.9.1/app-defaults/out/XPaint_fr
--- xpaint-2.9.9.1-orig/app-defaults/out/XPaint_fr	2012-02-08 11:51:15.000000000 +0100
+++ xpaint-2.9.9.1/app-defaults/out/XPaint_fr	2012-12-16 22:24:42.488302368 +0100
@@ -13,7 +13,7 @@ xpaint*msgFile:  messages/Messages_fr
 xpaint*menufont: Liberation Sans-9
 
 ! Xpaint share directory
-xpaint*shareDir: /usr/share/xpaint
+xpaint*shareDir: /kanapi_packages/xpaint-2.9.9.1/share/xpaint
 
 !  This fixes Xpaint's startup features
 !xpaint*horizontal: True
diff -urNp xpaint-2.9.9.1-orig/Local.config xpaint-2.9.9.1/Local.config
--- xpaint-2.9.9.1-orig/Local.config	2010-11-07 22:32:05.000000000 +0100
+++ xpaint-2.9.9.1/Local.config	2012-12-16 21:49:26.725809687 +0100
@@ -8,7 +8,7 @@ XCOMM $Id: Local.config,v 1.18 2005/03/2
 
 XCOMM Local customization
 
-DESTDIR = 
+DESTDIR =
 SHAREDIR = /usr/share/xpaint
 
 XCOMM Printing system defaults
diff -urNp xpaint-2.9.9.1-orig/Makefile.am xpaint-2.9.9.1/Makefile.am
--- xpaint-2.9.9.1-orig/Makefile.am	2012-03-01 21:12:56.000000000 +0100
+++ xpaint-2.9.9.1/Makefile.am	2012-12-16 22:25:25.120513063 +0100
@@ -3,7 +3,7 @@
 ## Local Configuration arguments
 BINDIR = $(prefix)/bin
 SHAREDIR = $(prefix)/share/xpaint
-XAPPLOADDIR = /etc/X11/app-defaults
+XAPPLOADDIR = $(prefix)/etc/X11/app-defaults
 
 ## Printing system defaults
 PRINT_COMMAND = lpr
@@ -18,7 +18,7 @@ LPCCMD = /usr/sbin/lpc status
 ## PRINTCAP = /etc/printcap:/etc/printcap.cups
 ## #define PRINTCAP 
 
-EDITOR = "emacs -fn 9x15 -cr green -ms red -bg lightyellow -fg black"
+EDITOR = "gvim"
 POSTSCRIPT_VIEWER = gv
 EXTERN_VIEWER = display
 
@@ -146,4 +146,4 @@ clean-local:
 
 distclean-local:
 	cd rw ; make distclean
-	cd util ; make distclean
\ Brak znaku nowej linii na końcu pliku
+	cd util ; make distclean
diff -urNp xpaint-2.9.9.1-orig/Makefile.in xpaint-2.9.9.1/Makefile.in
--- xpaint-2.9.9.1-orig/Makefile.in	2012-03-01 21:13:14.000000000 +0100
+++ xpaint-2.9.9.1/Makefile.in	2012-12-16 22:25:43.568604064 +0100
@@ -224,11 +224,11 @@ top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 BINDIR = $(prefix)/bin
 SHAREDIR = $(prefix)/share/xpaint
-XAPPLOADDIR = /etc/X11/app-defaults
+XAPPLOADDIR = $(prefix)/etc/X11/app-defaults
 PRINT_COMMAND = lpr
 LPCCMD = /usr/sbin/lpc status
 #define LPCCMD 
-EDITOR = "emacs -fn 9x15 -cr green -ms red -bg lightyellow -fg black"
+EDITOR = "gvim"
 POSTSCRIPT_VIEWER = gv
 EXTERN_VIEWER = display
 ARCH_DEFINES = -g -O2 -fno-strict-aliasing -fPIC
diff -urNp xpaint-2.9.9.1-orig/share/c_scripts/layers/combine.c xpaint-2.9.9.1/share/c_scripts/layers/combine.c
--- xpaint-2.9.9.1-orig/share/c_scripts/layers/combine.c	2005-03-27 21:54:50.000000000 +0200
+++ xpaint-2.9.9.1/share/c_scripts/layers/combine.c	2012-12-16 21:19:20.212852380 +0100
@@ -20,8 +20,8 @@ Image * ImageCreate()
     int x, y, dx, width, height;
 
     /* Open files */
-    image[0] = ImageFromFile("/usr/share/xpaint/XPaintIcon.xpm");
-    image[1] = ImageFromFile("/usr/share/xpaint/c_scripts/3d_surfaces/ellipsoid.c");
+    image[0] = ImageFromFile("/kanapi_packages/xpaint-2.9.9.1/usr/share/xpaint/XPaintIcon.xpm");
+    image[1] = ImageFromFile("/kanapi_packages/xpaint-2.9.9.1/usr/share/xpaint/c_scripts/3d_surfaces/ellipsoid.c");
 
     if (!image[0] || !image[1]) {
        fprintf(stderr, "Cannot find XPaintIcon.xpm or penguin.png !!\n");
diff -urNp xpaint-2.9.9.1-orig/util/Makefile xpaint-2.9.9.1/util/Makefile
--- xpaint-2.9.9.1-orig/util/Makefile	2012-02-05 15:14:24.000000000 +0100
+++ xpaint-2.9.9.1/util/Makefile	2012-12-16 21:51:32.902436059 +0100
@@ -1,7 +1,10 @@
 GCC = gcc
 CPP = g++ -fpermissive
 
-all: pdfconcat ppmtops pgf2pnm
+INSTALLDIR=/kanapi_packages/xpaint-2.9.9.1
+
+all: pdfconcat ppmtops
+#	pgf2pnm
 
 pdfconcat:
 	$(GCC) -O3 -s -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes -Wmissing-declarations pdfconcat.c -o pdfconcat
@@ -19,21 +22,22 @@ pgf2pnm: main.cpp pnm.cpp
 	fi
 
 install:
+	echo "install $(INSTALLDIR)"
 	@if [ -x pdfconcat ] ; then \
-	mkdir -p $(DESTDIR)/usr/bin ; \
-	install -c pdfconcat $(DESTDIR)/usr/bin ; \
-	echo "install -c pdfconcat $(DESTDIR)/usr/bin" ; \
-	mkdir -p $(DESTDIR)/usr/share/man/man1 ; \
-	install -c -m 644 pdfconcat.1 $(DESTDIR)/usr/share/man/man1 ; \
-	echo "install -c pdfconcat.1 $(DESTDIR)/usr/share/man/man1" ; fi
+	mkdir -p $(INSTALLDIR)/usr/bin ; \
+	install -c pdfconcat $(INSTALLDIR)/usr/bin ; \
+	echo "install -c pdfconcat $(INSTALLDIR)/usr/bin" ; \
+	mkdir -p $(INSTALLDIR)/usr/share/man/man1 ; \
+	install -c -m 644 pdfconcat.1 $(INSTALLDIR)/usr/share/man/man1 ; \
+	echo "install -c pdfconcat.1 $(INSTALLDIR)/usr/share/man/man1" ; fi
 	@if [ -x ppmtops ] ; then \
-	mkdir -p $(DESTDIR)/usr/bin ; \
-	install -c ppmtops $(DESTDIR)/usr/bin ; \
-	echo "install -c ppmtops $(DESTDIR)/usr/bin" ; fi
+	mkdir -p $(INSTALLDIR)/usr/bin ; \
+	install -c ppmtops $(INSTALLDIR)/usr/bin ; \
+	echo "install -c ppmtops $(INSTALLDIR)/usr/bin" ; fi
 	@if [ -x pgf2pnm ] ; then \
-	mkdir -p $(DESTDIR)/usr/bin ; \
-	install -c pgf2pnm $(DESTDIR)/usr/bin ; \
-	echo "install -c pgf2pnm $(DESTDIR)/usr/bin" ; fi
+	mkdir -p $(INSTALLDIR)/usr/bin ; \
+	install -c pgf2pnm $(INSTALLDIR)/usr/bin ; \
+	echo "install -c pgf2pnm $(INSTALLDIR)/usr/bin" ; fi
 
 clean:
 	rm -f core *~ *.o pdfconcat ppmtops pgf2pnm
diff -urNp xpaint-2.9.9.1-orig/XPaint.ad xpaint-2.9.9.1/XPaint.ad
--- xpaint-2.9.9.1-orig/XPaint.ad	2012-02-08 11:51:15.000000000 +0100
+++ xpaint-2.9.9.1/XPaint.ad	2012-12-16 22:26:40.144885686 +0100
@@ -13,7 +13,7 @@ xpaint*msgFile:  messages/Messages
 xpaint*menufont: Liberation-10:matrix=0.85 0 0 0.9
 
 ! Xpaint share directory
-xpaint*shareDir: /usr/share/xpaint
+xpaint*shareDir: /kanapi_packages/xpaint-2.9.9.1/share/xpaint
 
 !  This fixes Xpaint's startup features
 !xpaint*horizontal: True
