diff -urNp gnome-keyring-2.32.1-orig/egg/egg-asn1x.c gnome-keyring-2.32.1/egg/egg-asn1x.c
--- gnome-keyring-2.32.1-orig/egg/egg-asn1x.c	2010-10-19 02:11:38.000000000 +0000
+++ gnome-keyring-2.32.1/egg/egg-asn1x.c	2013-01-30 09:05:11.624012615 +0000
@@ -46,10 +46,10 @@
  */
 
 #include "config.h"
+#include <libtasn1.h>
 
 #include "egg-asn1x.h"
 
-#include <libtasn1.h>
 
 #include <stdlib.h>
 #include <string.h>
