diff -urNp libmad-0.15.1b-orig/configure libmad-0.15.1b/configure
--- libmad-0.15.1b-orig/configure	2004-02-05 09:34:07.000000000 +0000
+++ libmad-0.15.1b/configure	2012-12-29 20:55:53.512287204 +0000
@@ -19099,7 +19099,6 @@ then
     case "$optimize" in
 	-O|"-O "*)
 	    optimize="-O"
-	    optimize="$optimize -fforce-mem"
 	    optimize="$optimize -fforce-addr"
 	    : #x optimize="$optimize -finline-functions"
 	    : #- optimize="$optimize -fstrength-reduce"
