diff -urN net-tools-1.60-orig/hostname.c net-tools-1.60/hostname.c
--- net-tools-1.60-orig/hostname.c	2001-04-08 19:04:23.000000000 +0200
+++ net-tools-1.60/hostname.c	2012-09-11 00:46:16.000000000 +0200
@@ -97,7 +97,7 @@
 	case EINVAL:
 	    fprintf(stderr, _("%s: name too long\n"), program_name);
 	    break;
-	default:
+	default:;
 	}
 	exit(1);
     };
@@ -116,7 +116,7 @@
 	case EINVAL:
 	    fprintf(stderr, _("%s: name too long\n"), program_name);
 	    break;
-	default:
+	default:;
 	}
 	exit(1);
     };
@@ -173,7 +173,7 @@
 	    *p = '\0';
 	printf("%s\n", hp->h_name);
 	break;
-    default:
+    default:;
     }
 }
 
diff -urN net-tools-1.60-orig/lib/inet_sr.c net-tools-1.60/lib/inet_sr.c
--- net-tools-1.60-orig/lib/inet_sr.c	2000-02-20 22:46:45.000000000 +0100
+++ net-tools-1.60/lib/inet_sr.c	2012-09-11 00:42:45.000000000 +0200
@@ -105,6 +105,7 @@
     case 2:
        isnet = 0; break;
     default:
+       break;
     }
 
     /* Fill in the other fields. */
