diff -urNp xfce4-notes-plugin-1.7.7-orig/src/main-status-icon.c xfce4-notes-plugin-1.7.7/src/main-status-icon.c
--- xfce4-notes-plugin-1.7.7-orig/src/main-status-icon.c	2010-11-07 11:28:38.000000000 +0000
+++ xfce4-notes-plugin-1.7.7/src/main-status-icon.c	2013-01-21 00:37:37.165912773 +0000
@@ -125,7 +125,7 @@ void build_plugin (void) {
 	application = (_tmp0_ = xnp_application_new (save_location), _g_object_unref0 (application), _tmp0_);
 	status_icon = (_tmp1_ = gtk_status_icon_new_from_icon_name ("xfce4-notes-plugin"), _g_object_unref0 (status_icon), _tmp1_);
 	gtk_status_icon_set_tooltip (status_icon, _ ("Notes"));
-	g_timeout_add_seconds_full (G_PRIORITY_DEFAULT, (guint) 60, __lambda0__gsource_func, NULL, NULL);
+//	g_timeout_add_seconds_full (G_PRIORITY_DEFAULT, (guint) 60, __lambda0__gsource_func, NULL, NULL);
 	g_signal_connect (status_icon, "activate", (GCallback) __lambda1__gtk_status_icon_activate, NULL);
 	context_menu = (_tmp2_ = build_context_menu (), _g_object_unref0 (context_menu), _tmp2_);
 	g_signal_connect (status_icon, "popup-menu", (GCallback) __lambda5__gtk_status_icon_popup_menu, NULL);
