diff -urNp graphviz-2.30.1-orig/lib/cdt/cdt.h graphviz-2.30.1/lib/cdt/cdt.h
--- graphviz-2.30.1-orig/lib/cdt/cdt.h	2013-02-14 13:27:39.000000000 +0000
+++ graphviz-2.30.1/lib/cdt/cdt.h	2013-02-14 23:37:06.346173770 +0000
@@ -239,8 +239,8 @@ extern int		dtstat _ARG_((Dt_t*, Dtstat_
 extern unsigned int	dtstrhash _ARG_((unsigned int, Void_t*, int));
 
 #if !_PACKAGE_ast
-extern int		memcmp _ARG_((const Void_t*, const Void_t*, size_t));
-extern int		strcmp _ARG_((const char*, const char*));
+//extern int		memcmp _ARG_((const Void_t*, const Void_t*, size_t));
+//extern int		strcmp _ARG_((const char*, const char*));
 #endif
 
 #undef extern
diff -urNp graphviz-2.30.1-orig/lib/common/arith.h graphviz-2.30.1/lib/common/arith.h
--- graphviz-2.30.1-orig/lib/common/arith.h	2013-02-14 13:27:39.000000000 +0000
+++ graphviz-2.30.1/lib/common/arith.h	2013-02-14 23:41:37.637180852 +0000
@@ -91,7 +91,7 @@ extern "C" {
 #define SQR(a) ((a) * (a))
 
 #ifdef HAVE_SINCOS
-    extern void sincos(double x, double *s, double *c);
+//    extern void sincos(double x, double *s, double *c);
 #else
 # define sincos(x,s,c) *s = sin(x); *c = cos(x)
 #endif
