diff -urNp dia-0.97.2-orig/objects/custom/shape_typeinfo.c dia-0.97.2/objects/custom/shape_typeinfo.c
--- dia-0.97.2-orig/objects/custom/shape_typeinfo.c	2009-11-07 14:28:34.000000000 +0000
+++ dia-0.97.2/objects/custom/shape_typeinfo.c	2013-02-24 22:12:38.942220443 +0000
@@ -27,8 +27,7 @@
 #include "custom_util.h"
 #include <string.h>
 #include <stdarg.h>
-#include <glib/gstrfuncs.h>
-#include <glib/gstdio.h>
+#include <glib.h>
 #include <libxml/parser.h>
 
 /*
diff -urNp dia-0.97.2-orig/tests/test-boundingbox.c dia-0.97.2/tests/test-boundingbox.c
--- dia-0.97.2-orig/tests/test-boundingbox.c	2009-11-07 14:28:35.000000000 +0000
+++ dia-0.97.2/tests/test-boundingbox.c	2013-02-24 22:23:37.002237480 +0000
@@ -28,9 +28,11 @@
 #include <glib.h>
 #include <glib-object.h>
 
+#if 0
 #if GLIB_CHECK_VERSION(2,16,0)
 #include <glib/gtestutils.h>
 #endif
+#endif
 #include "dialib.h"
 
 /*
diff -urNp dia-0.97.2-orig/tests/test-objects.c dia-0.97.2/tests/test-objects.c
--- dia-0.97.2-orig/tests/test-objects.c	2009-11-07 14:28:35.000000000 +0000
+++ dia-0.97.2/tests/test-objects.c	2013-02-24 22:24:51.341239414 +0000
@@ -28,10 +28,11 @@
 #include <glib.h>
 #include <glib-object.h>
 
+#if 0
 #if GLIB_CHECK_VERSION(2,16,0)
 #include <glib/gtestutils.h>
 #endif
-
+#endif
 #include "object.h"
 #include "plug-ins.h"
 #include "dialib.h"
