diff -Nurp smart-1.4.1-orig/smart/const.py smart-1.4.1/smart/const.py
--- smart-1.4.1-orig/smart/const.py	2011-10-09 21:40:34.000000000 +0000
+++ smart-1.4.1/smart/const.py	2013-06-13 22:36:55.630830465 +0000
@@ -66,8 +66,8 @@ BLOCKSIZE = 16384
 
 DISTROFILE  = "/usr/lib/smart/distro.py"
 PLUGINSDIR  = "/usr/lib/smart/plugins/"
-DATADIR     = "/var/lib/smart/"
-USERDATADIR = "~/.smart/"
+DATADIR     = "/kanapi_packages/smart-1.4.1/var/data/"
+USERDATADIR = "/kanapi_packages/smart-1.4.1/var/user/"
 CONFFILE    = "config"
 
 # vim:ts=4:sw=4:et
