diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindBoost.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindBoost.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindBoost.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindBoost.cmake	2013-01-12 17:03:22.152509075 +0000
@@ -600,11 +600,9 @@ set(Boost_ERROR_REASON)
   endif(WIN32)
 
   set(_boost_INCLUDE_SEARCH_DIRS_SYSTEM
-    C:/boost/include
-    C:/boost
     "$ENV{ProgramFiles}/boost/include"
     "$ENV{ProgramFiles}/boost"
-    /sw/local/include
+    /kanapi_packages/boost-1_52_0/include/boost
   )
 
   _Boost_CHECK_SPELLING(Boost_ROOT)
@@ -863,7 +861,7 @@ set(Boost_ERROR_REASON)
     "$ENV{ProgramFiles}/boost/boost_${Boost_MAJOR_VERSION}_${Boost_MINOR_VERSION}/lib"
     "$ENV{ProgramFiles}/boost/lib"
     "$ENV{ProgramFiles}/boost"
-    /sw/local/lib
+    /kanapi_packages/boost-1_52_0/lib
   )
   set(_boost_LIBRARY_SEARCH_DIRS ${_boost_LIBRARY_SEARCH_DIRS_ALWAYS})
   if( Boost_NO_SYSTEM_PATHS )
diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindFreetype.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindFreetype.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindFreetype.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindFreetype.cmake	2013-01-12 10:33:59.213585997 +0000
@@ -46,19 +46,14 @@ FIND_PATH(FREETYPE_INCLUDE_DIR_ft2build
   HINTS
   $ENV{FREETYPE_DIR}
   PATHS
-  /usr/local/X11R6/include
-  /usr/local/X11/include
-  /usr/freeware/include
+  /kanapi_packages/freetype-2.4.10/include
 )
 
 FIND_PATH(FREETYPE_INCLUDE_DIR_freetype2 freetype/config/ftheader.h
   HINTS
   $ENV{FREETYPE_DIR}/include/freetype2
   PATHS
-  /usr/local/X11R6/include
-  /usr/local/X11/include
-  /usr/freeware/include
-  PATH_SUFFIXES freetype2
+  /kanapi_packages/freetype-2.4.10/include/freetype2
 )
 
 FIND_LIBRARY(FREETYPE_LIBRARY
@@ -67,9 +62,7 @@ FIND_LIBRARY(FREETYPE_LIBRARY
   $ENV{FREETYPE_DIR}
   PATH_SUFFIXES lib64 lib
   PATHS
-  /usr/local/X11R6
-  /usr/local/X11
-  /usr/freeware
+  /kanapi_packages/freetype-2.4.10/lib
 )
 
 # set the user variables
diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindOpenAL.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindOpenAL.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindOpenAL.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindOpenAL.cmake	2013-01-12 17:15:23.274493861 +0000
@@ -66,13 +66,7 @@ FIND_PATH(OPENAL_INCLUDE_DIR al.h
   $ENV{OPENALDIR}
   PATH_SUFFIXES include/AL include/OpenAL include
   PATHS
-  ~/Library/Frameworks
-  /Library/Frameworks
-  /sw # Fink
-  /opt/local # DarwinPorts
-  /opt/csw # Blastwave
-  /opt
-  [HKEY_LOCAL_MACHINE\\SOFTWARE\\Creative\ Labs\\OpenAL\ 1.1\ Software\ Development\ Kit\\1.00.0000;InstallDir]
+  /kanapi_packages/openal-soft-1.15.1/include/AL
 )
 
 FIND_LIBRARY(OPENAL_LIBRARY 
@@ -81,13 +75,7 @@ FIND_LIBRARY(OPENAL_LIBRARY
   $ENV{OPENALDIR}
   PATH_SUFFIXES lib64 lib libs64 libs libs/Win32 libs/Win64
   PATHS
-  ~/Library/Frameworks
-  /Library/Frameworks
-  /sw
-  /opt/local
-  /opt/csw
-  /opt
-  [HKEY_LOCAL_MACHINE\\SOFTWARE\\Creative\ Labs\\OpenAL\ 1.1\ Software\ Development\ Kit\\1.00.0000;InstallDir]
+  /kanapi_packages/openal-soft-1.15.1/lib
 )
 
 
diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindOpenGL.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindOpenGL.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindOpenGL.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindOpenGL.cmake	2013-01-12 09:17:24.317030117 +0000
@@ -77,9 +77,7 @@ ELSE (WIN32)
     #  - Atanas Georgiev <atanas@cs.columbia.edu>
 
     FIND_PATH(OPENGL_INCLUDE_DIR GL/gl.h
-      /usr/share/doc/NVIDIA_GLX-1.0/include
-      /usr/openwin/share/include
-      /opt/graphics/OpenGL/include /usr/X11R6/include
+      /kanapi_packages/Mesa-8.0.4/include
     )
 
     FIND_PATH(OPENGL_xmesa_INCLUDE_DIR GL/xmesa.h
@@ -90,10 +88,7 @@ ELSE (WIN32)
 
     FIND_LIBRARY(OPENGL_gl_LIBRARY
       NAMES GL MesaGL
-      PATHS /opt/graphics/OpenGL/lib
-            /usr/openwin/lib
-            /usr/shlib /usr/X11R6/lib
-            ${HPUX_IA_OPENGL_LIB_PATH}
+      PATHS /kanapi_packages/Mesa-8.0.4/lib
     )
 
     # On Unix OpenGL most certainly always requires X11.
diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindPNG.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindPNG.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindPNG.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindPNG.cmake	2013-01-12 09:42:42.785046704 +0000
@@ -35,7 +35,7 @@ find_package(ZLIB ${_FIND_ZLIB_ARG})
 
 if(ZLIB_FOUND)
   find_path(PNG_PNG_INCLUDE_DIR png.h
-  /usr/local/include/libpng             # OpenBSD
+	  /kanapi_packages/libpng-1.5.13/include/libpng15
   )
 
   set(PNG_NAMES ${PNG_NAMES} png libpng png15 libpng15 png15d libpng15d png14 libpng14 png14d libpng14d png12 libpng12 png12d libpng12d)
diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindSDL.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindSDL.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindSDL.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindSDL.cmake	2013-01-12 14:13:20.257050370 +0000
@@ -70,16 +70,7 @@ FIND_PATH(SDL_INCLUDE_DIR SDL.h
   $ENV{SDLDIR}
   PATH_SUFFIXES include/SDL include
   PATHS
-  ~/Library/Frameworks
-  /Library/Frameworks
-  /usr/local/include/SDL12
-  /usr/local/include/SDL11 # FreeBSD ports
-  /usr/include/SDL12
-  /usr/include/SDL11
-  /sw # Fink
-  /opt/local # DarwinPorts
-  /opt/csw # Blastwave
-  /opt
+  /kanapi_packages/SDL-1.2.15/include/SDL
 )
 
 # SDL-1.1 is the name used by FreeBSD ports...
@@ -90,10 +81,7 @@ FIND_LIBRARY(SDL_LIBRARY_TEMP
   $ENV{SDLDIR}
   PATH_SUFFIXES lib64 lib
   PATHS
-  /sw
-  /opt/local
-  /opt/csw
-  /opt
+  /kanapi_packages/SDL-1.2.15/lib
 )
 
 IF(NOT SDL_BUILDING_LIBRARY)
@@ -108,10 +96,7 @@ IF(NOT SDL_BUILDING_LIBRARY)
       $ENV{SDLDIR}
       PATH_SUFFIXES lib64 lib
       PATHS
-      /sw
-      /opt/local
-      /opt/csw
-      /opt
+      /kanapi_packages/SDL-1.2.15/lib
     )
   ENDIF(NOT ${SDL_INCLUDE_DIR} MATCHES ".framework")
 ENDIF(NOT SDL_BUILDING_LIBRARY)
diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindSDL_image.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindSDL_image.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindSDL_image.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindSDL_image.cmake	2013-01-12 14:13:57.694582325 +0000
@@ -31,22 +31,7 @@ FIND_PATH(SDLIMAGE_INCLUDE_DIR SDL_image
   $ENV{SDLDIR}
   PATH_SUFFIXES include
   PATHS
-  ~/Library/Frameworks
-  /Library/Frameworks
-  /usr/local/include/SDL
-  /usr/include/SDL
-  /usr/local/include/SDL12
-  /usr/local/include/SDL11 # FreeBSD ports
-  /usr/include/SDL12
-  /usr/include/SDL11
-  /sw/include/SDL # Fink
-  /sw/include
-  /opt/local/include/SDL # DarwinPorts
-  /opt/local/include
-  /opt/csw/include/SDL # Blastwave
-  /opt/csw/include 
-  /opt/include/SDL
-  /opt/include
+  /kanapi_packages/SDL_image-1.2.12/include/SDL
 )
 
 FIND_LIBRARY(SDLIMAGE_LIBRARY 
@@ -56,12 +41,7 @@ FIND_LIBRARY(SDLIMAGE_LIBRARY
   $ENV{SDLDIR}
   PATH_SUFFIXES lib64 lib
   PATHS
-  ~/Library/Frameworks
-  /Library/Frameworks
-  /sw
-  /opt/local
-  /opt/csw
-  /opt
+  /kanapi_packages/SDL_image-1.2.12/lib
 )
 
 INCLUDE(${CMAKE_CURRENT_LIST_DIR}/FindPackageHandleStandardArgs.cmake)
diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindSDL_sound.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindSDL_sound.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindSDL_sound.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindSDL_sound.cmake	2013-01-12 10:37:27.384983504 +0000
@@ -263,10 +263,7 @@ IF(SDL_FOUND AND SDL_SOUND_INCLUDE_DIR A
          $ENV{SDLSOUNDDIR}
          $ENV{SDLDIR}/lib
          $ENV{SDLDIR}
-         /sw/lib
-         /opt/local/lib
-         /opt/csw/lib
-       /opt/lib
+       /kanapi_packages/libogg-1.3.0/lib
          )
        IF(VORBIS_LIBRARY)
          SET(SDL_SOUND_LIBRARIES_TMP ${SDL_SOUND_LIBRARIES_TMP} ${VORBIS_LIBRARY})
@@ -283,10 +280,7 @@ IF(SDL_FOUND AND SDL_SOUND_INCLUDE_DIR A
          $ENV{SDLSOUNDDIR}
          $ENV{SDLDIR}/lib
          $ENV{SDLDIR}
-         /sw/lib
-         /opt/local/lib
-         /opt/csw/lib
-       /opt/lib
+       /kanapi_packages/libvorbis-1.3.3/lib
          )
        IF(OGG_LIBRARY)
          SET(SDL_SOUND_LIBRARIES_TMP ${SDL_SOUND_LIBRARIES_TMP} ${OGG_LIBRARY})
diff -urNp cmake-2.8.9-orig/share/cmake-2.8/Modules/FindX11.cmake cmake-2.8.9/share/cmake-2.8/Modules/FindX11.cmake
--- cmake-2.8.9-orig/share/cmake-2.8/Modules/FindX11.cmake	2012-08-09 18:15:19.000000000 +0000
+++ cmake-2.8.9/share/cmake-2.8/Modules/FindX11.cmake	2013-01-12 10:15:55.278137110 +0000
@@ -61,20 +61,37 @@ IF (UNIX)
   SET(CMAKE_FIND_FRAMEWORK_SAVE ${CMAKE_FIND_FRAMEWORK})
   SET(CMAKE_FIND_FRAMEWORK NEVER)
   SET(X11_INC_SEARCH_PATH
-    /usr/pkg/xorg/include
-    /usr/X11R6/include
-    /usr/X11R7/include
-    /usr/include/X11
-    /usr/openwin/include
-    /usr/openwin/share/include
-    /opt/graphics/OpenGL/include
+    /kanapi_packages/libX11-1.5.0/include
+    /kanapi_packages/libxcb-1.8.1/include
+    /kanapi_packages/xproto-7.0.23/include
+    /kanapi_packages/kbproto-1.0.6/include
+    /kanapi_packages/libXau-1.0.7/include
+    /kanapi_packages/libXt-1.1.3/include
+    /kanapi_packages/libICE-1.0.8/include
+    /kanapi_packages/libSM-1.2.1/include
+    /kanapi_packages/xproto-7.0.23/include 
+    /kanapi_packages/libxcb-1.8.1/include 
+    /kanapi_packages/libXau-1.0.7/include 
+    /kanapi_packages/libXext-1.3.1/include
+    /kanapi_packages/xextproto-7.2.1/include
+    /kanapi_packages/libXcomposite-0.4.3/include
+    /kanapi_packages/libXfixes-5.0/include
+    /kanapi_packages/compositeproto-0.4.2/include
+    /kanapi_packages/fixesproto-5.0/include
+    /kanapi_packages/libXrandr-1.3.2/include
+    /kanapi_packages/libXrender-0.9.7/include
   )
 
   SET(X11_LIB_SEARCH_PATH
-    /usr/pkg/xorg/lib
-    /usr/X11R6/lib
-    /usr/X11R7/lib
-    /usr/openwin/lib
+  /kanapi_packages/libX11-1.5.0/lib
+  /kanapi_packages/libXt-1.1.3/lib
+  /kanapi_packages/libICE-1.0.8/lib
+  /kanapi_packages/libSM-1.2.1/lib
+  /kanapi_packages/libXext-1.3.1/lib
+  /kanapi_packages/libXau-1.0.7/lib
+  /kanapi_packages/libXcomposite-0.4.3/lib
+  /kanapi_packages/libXrandr-1.3.2/lib
+  /kanapi_packages/libXrender-0.9.7/lib
   )
 
   FIND_PATH(X11_X11_INCLUDE_PATH X11/X.h                             ${X11_INC_SEARCH_PATH})
