diff -urN sge030809-orig/Makefile.conf sge030809/Makefile.conf
--- sge030809-orig/Makefile.conf	2003-08-12 00:43:35.000000000 +0200
+++ sge030809/Makefile.conf	2012-12-04 01:21:00.666489939 +0100
@@ -3,8 +3,8 @@
 # Comment/uncomment the following line to disable/enable build options
 # (See README for more info)
 C_COMP = y
-#USE_FT = n
-#USE_IMG = n
+USE_FT = n
+USE_IMG = y
 #QUIET = y
 
 
@@ -19,20 +19,20 @@
 endif
 
 # Where should SGE be installed?
-PREFIX =$(shell sdl-config --prefix)
+PREFIX =/kanapi_packages/sge030809
 
 # Where should the headerfiles be installed?
-PREFIX_H =$(shell sdl-config --prefix)/include/SDL
+PREFIX_H =/kanapi_packages/sge030809/include/SDL
 
 # Flags passed to the compiler
-CFLAGS =-Wall -O3 -ffast-math
-SGE_CFLAGS =$(shell sdl-config --cflags)
+CFLAGS =-Wall -O3 -ffast-math -mtune=native
+SGE_CFLAGS =$(shell sdl-config --cflags) -I/kanapi_packages/SDL_image-1.2.12/include/SDL
 # Uncomment to make some more optimizations
 #CFLAGS =-Wall -O9 -ffast-math -march=i686
 
 
 # Libs config
-SGE_LIBS =$(shell sdl-config --libs) -lstdc++
+SGE_LIBS =$(shell sdl-config --libs) -lstdc++ -L/kanapi_packages/SDL_image-1.2.12/lib
 
 
 # Is freetype-config available?
