diff -ur src-orig/osdef2.h.in src/osdef2.h.in
--- src-orig/osdef2.h.in	2010-05-15 13:04:08.000000000 +0200
+++ src/osdef2.h.in	2012-12-15 23:54:09.270332230 +0100
@@ -53,11 +53,11 @@
 extern int	mkdir __ARGS((const char *, mode_t));
 extern int	rmdir __ARGS((const char *));
 
-extern int	tgetent __ARGS((char *, char *));
+extern int	tgetent __ARGS((char *, const char *));
 extern int	tgetnum __ARGS((char *));
 extern int	tgetflag __ARGS((char *));
-extern char	*tgoto __ARGS((char *, int, int));
-extern int	tputs __ARGS((char *, int, int (*)(int)));
+extern char	*tgoto __ARGS((const char *, int, int));
+extern int	tputs __ARGS((const char *, int, int (*)(int)));
 
 #ifdef HAVE_TERMIOS_H
 struct termios;		/* for tcgetattr __ARGS */
@@ -92,5 +92,5 @@
 extern int	_Xmblen __ARGS((char const *, size_t));
 #else
 		/* This is different from the header but matches mblen() */
-extern int	_Xmblen __ARGS((char *, size_t));
+extern int	_Xmblen __ARGS((char *, int));
 #endif
