diff -urN rplay-3.3.2-orig/rplayd/misc.c rplay-3.3.2/rplayd/misc.c
--- rplay-3.3.2-orig/rplayd/misc.c	1999-03-10 08:58:03.000000000 +0100
+++ rplay-3.3.2/rplayd/misc.c	2012-09-29 00:45:23.404416138 +0200
@@ -42,7 +42,7 @@
 #include <math.h>
 #include "sound.h"
 #include "timer.h"
-
+#include <errno.h>
 #ifdef __STDC__
 char *
 sys_err_str(int error)
diff -urN rplay-3.3.2-orig/rplayd/rplayd.c rplay-3.3.2/rplayd/rplayd.c
--- rplay-3.3.2-orig/rplayd/rplayd.c	1999-03-10 08:58:04.000000000 +0100
+++ rplay-3.3.2/rplayd/rplayd.c	2012-09-29 00:41:47.451345234 +0200
@@ -186,7 +186,7 @@
 #endif /* OTHER_RPLAY_PORTS */
 static int audio_test_mode = 0;	/* enable audio test mode */
 static int do_fork = 1;
-static int inetd = 0;		/* Was rplayd started by inetd? */
+int inetd = 0;		/* Was rplayd started by inetd? */
 
 #ifndef RPLAYD_USER
 #define RPLAYD_USER ""
diff -urN rplay-3.3.2-orig/rplayd/timer.c rplay-3.3.2/rplayd/timer.c
--- rplay-3.3.2-orig/rplayd/timer.c	1999-03-10 08:58:04.000000000 +0100
+++ rplay-3.3.2/rplayd/timer.c	2012-09-29 00:46:08.132638325 +0200
@@ -31,6 +31,7 @@
 #include <unistd.h>
 #include "rplayd.h"
 #include "timer.h"
+#include <errno.h>
 
 double timer_rate = 0.0;
 double timer_count = 0.0;
