diff -urNp SDL_net-1.2.8-orig/chat.cpp SDL_net-1.2.8/chat.cpp
--- SDL_net-1.2.8-orig/chat.cpp	2012-01-15 16:20:10.000000000 +0000
+++ SDL_net-1.2.8/chat.cpp	2013-01-03 19:50:07.671480754 +0000
@@ -17,7 +17,7 @@
   2. Altered source versions must be plainly marked as such, and must not be
      misrepresented as being the original software.
   3. This notice may not be removed or altered from any source distribution.
-*/
+
     Pleasanton, CA 94588 (USA)
     slouken@devolution.com
 */
